### DISCUSSION TITLE
Description field for media content type

### DISCUSSION THREAD

**Discussion by InSecker** [strapi/strapi#7922] [Legacy Topics]
Hi there :)

I want to add an extra field in the "View Edit" page for media type. My need is to give to user reference image ratio for uploading.

I'm a bit confused with all the different packages and I can't find where I can overload the basic media type.

**Comment by derrickmehaffy** [comment]
Hello, take a look at this guide to see if it will help you :smiley: 
https://strapi.io/documentation/v3.x/guides/registering-a-field-in-admin.html#introduction
